src/lsm/forest_fuzz.zig

```diff
@@ -535,28 +585,57 @@ pub fn generate_fuzz_ops(random: std.rand.Random, fuzz_op_count: usize) ![]const
                 account.credits_posted = random.int(u64);
 
                 try id_to_account.put(account.id, account);
-                break :put_account FuzzOp{ .put_account = .{
+                break :put_account FuzzOpAction{ .put_account = .{
                     .op = op,
                     .account = account,
                 } };
             },
-            .get_account => FuzzOp{ .get_account = random_id(random, u128) },
-            .storage_reset => storage_reset: {
-                op = persisted_op;
-                break :storage_reset FuzzOp{ .storage_reset = {} };
-            },
+            .get_account => FuzzOpAction{ .get_account = random_id(random, u128) },
         };
-        switch (fuzz_op.*) {
+        switch (action) {
             .compact => puts_since_compact = 0,
             .put_account => puts_since_compact += 1,
             .get_account => {},
-            .storage_reset => {},
         }
+        // TODO(jamii)
+        // Currently, crashing is only interesting during a compact.
+        // But once we have concurrent compaction, crashing at any point can be interesting.
+        //
+        // TODO(jamii)
+        // If we crash during a checkpoint, on restart we should either:
+        // * See the state from that checkpoint.
+        // * See the state from the previous checkpoint.
+        // But this is difficult to test, so for now we'll avoid it.
+        const modifer_tag =
+            if (action == .compact and !action.compact.checkpoint)
+            fuzz.random_enum(
+                random,
+                FuzzOpModifierTag,
+                modifier_distribution,
+            )
+        else
+            FuzzOpModifierTag.normal;
```

Comment:
The indentation on this if branch is weird, but I see that zig fmt thinks that's the right way to do it. I think it'd be more readable with a block even though it's a single expression.

Reply:
Oh weird. I'll just push the if up onto the previous line.